resources/views/admin/layout/menu.blade.php

```diff
@@ -0,0 +1,31 @@
+<?php ?>
+<div class="navbar-default navbar-static-side" role="navigation">
+    <div class="sidebar-collapse">
+        <ul class="nav" id="side-menu">
+            <li>
+                <a href="/admin/user/list">
+                    <i class="fa fa-dashboard fa-fw"></i> Dashboard
+                </a>
+            </li>
+            <li>
+                <a href="/admin/user/list">
+                    <i class="fa fa-table fa-fw"></i> User 
+                    <span class="fa arrow"></span></a>
+                </a>
+                <ul class="nav nav-second-level">
+                    <li>
+                        <a href="/admin/users">List</a>
```

Comment:
/list and not /list , are they the same ? why there is 2 different link for same work ?

Reply:
I fixed, use /admin/user/list show list user